ros2topic/ros2topic/verb/info.py

```diff
@@ -39,7 +45,26 @@ def main(self, *, args):
                     break
             else:
                 return "Unknown topic '%s'" % topic_name
+
+            lineend = '\n'
```

Comment:
nitpick: (super nitpicky, feel free to ignore) I would have used `line_end`, as `lineend` is not a word.

Reply:
No problem. Done.